### ISSUE TITLE
Feature request: Ability to turn off 'top' functionality.

### ISSUE THREAD

**Issue by virtadpt** [htop-dev/htop#335]
How difficult would it be to add a hotkey or a command line option to disable the process top feature?  Here's the use case I have in mind:

I have a server which is sizable - 24 cores - and I use htop in a screen session to keep an eye on it (and keep my SSH session alive when I'm logged in). However, the user interface is a bit cramped no matter how I configure it, save if I maximize the terminal window I have open.  What I would like to do is this: Start one instance of htop which monitors the CPUs, memory, swap, time, and uptime but no process top.  I would like to start a second instance of htop that only shows the process top in a second screen window.

**Comment by BenBE** [comment]
You mean like #214 ???

**Comment by virtadpt** [comment]
No, I mean "go into the htop setup window, go to Display options, un-check the Show process list option," not pausing the process list.

**Comment by shirotech** [comment]
+1 for this feature, there are many use-cases to be able to properly turn off (hide) the process list.
